callouts/python/extproc/example/cloud_log/service_callout_example.py

```diff
@@ -49,7 +58,16 @@ def on_request_headers(
   def on_request_body(
       self, body: service_pb2.HttpBody, context: ServicerContext
   ) -> service_pb2.BodyResponse:
-    """Custom processor on the request body."""
+    """Custom processor on response headers.
+
+    Args:
+      headers (service_pb2.HttpHeaders): The HTTP headers received in the response.
```

Comment:
headers -> body (service_pb2.BodyResponse): The HTTP body received in the request.